aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -3,20 +3,50 @@
  * @copyright: http://www.divio.ch
  */
 
-'use strict';
+(function () {
+    'use strict';
 
-// #####################################################################################################################
-// #NAMESPACES#
-/* global Cl, describe, it, expect */
+    // ########################################################################
+    /* global Cl, describe, it, expect */
 
-// #####################################################################################################################
-// #UNIT TEST#
-describe('cl.events.js', function () {
-    it('Cl namespace is available', function () {
-        expect(Cl).toBeDefined();
-    });
+    // ########################################################################
+    describe('cl.events.js', function () {
+        beforeEach(function () {
+            fixture.setBase('frontend/fixtures');
+            this.markup = fixture.load('calendar.html');
+        });
+
+        afterEach(function(){
+            fixture.cleanup();
+        });
+
+        it('Cl namespace is available', function () {
```

Comment:
we probably need to change the naming here "cl.events.js Cl namespace is available" does not sound right